pkgs/by-name/py/pyprland/package.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, python311 }:
+
+python311.pkgs.buildPythonApplication rec {
+  pname = "pyprland";
+  version = "1.4.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "hyprland-community";
+    repo = "pyprland";
+    rev = version;
+    hash = "sha256-x/Rar80jwBX64pW+uv0edhlC44OP1b1e2vnJLFGlIms=";
+  };
+
+  nativeBuildInputs = with python311.pkgs; [ poetry-core ];
+
+  meta = with lib; {
+    description = "An hyperland plugin system";
+    homepage = "https://github.com/hyprland-community/pyprland";
+    license = licenses.mit;
+    maintainers = with maintainers; [ iliayar ];
```

```suggestion
    maintainers = with maintainers; [ iliayar ];
    mainProgram = "pypr";
```

`meta.mainProgram` should be set when there is a main binary